Lib/test/test_thread.py

```diff
@@ -32,6 +32,11 @@ def setUp(self):
         self.created = 0
         self.running = 0
         self.next_ident = 0
+        self.thread_count = thread._count()
+
+    def tearDown(self):
```

Comment:
This code looks similar to threading_cleanup(): please use threading_setup() and threading_cleanup() from test.support.